lib/interactivity-api.php

```diff
@@ -8,33 +8,10 @@
 /**
  * Deregisters the Core Interactivity API Modules and replace them
  * with the ones from the Gutenberg plugin.
+ *
+ * @deprecated 19.3.0 Script module registration is handled by {@see gutenberg_default_script_modules()}.
  */
-function gutenberg_reregister_interactivity_script_modules() {
-	$default_version = defined( 'GUTENBERG_VERSION' ) && ! SCRIPT_DEBUG ? GUTENBERG_VERSION : time();
-	wp_deregister_script_module( '@wordpress/interactivity' );
-	wp_deregister_script_module( '@wordpress/interactivity-router' );
-
-	wp_register_script_module(
-		'@wordpress/interactivity',
-		gutenberg_url( '/build-module/' . ( SCRIPT_DEBUG ? 'interactivity/debug.min.js' : 'interactivity/index.min.js' ) ),
-		array(),
-		$default_version
-	);
-
-	wp_register_script_module(
-		'@wordpress/interactivity-router',
-		gutenberg_url( '/build-module/interactivity-router/index.min.js' ),
-		array(
-			array(
-				'id'     => '@wordpress/a11y',
-				'import' => 'dynamic',
-			),
-			'@wordpress/interactivity',
-		),
-		$default_version
-	);
-}
-add_action( 'init', 'gutenberg_reregister_interactivity_script_modules' );
+function gutenberg_reregister_interactivity_script_modules() {}
```

Comment:
Fine to remove. We don't expect Gutenberg specific functions are used outside and we never considered it public API.

Reply:
Done, thanks 👍